docs/UserGuide.md

```diff
@@ -138,13 +138,25 @@ Format: `search [n/NAME] [i/ID] [g/SEX] [s/SPECIES] [b/BREED] [db/DOB] [da/DOA]`
 * Searches for animals whose attributes contain the specified search values in the prefix filter.
 * The search is case-insensitive. e.g `n/pookie` will match animals named `Pookie`
 * The order of prefixes does not matter. e.g. both `n/Pookie i/1234` and `i/1234 n/Pookie `will match animals named `Pookie` with ID `1234`
-* There must be at least 1 prefix specified.
+* Searching for `n/ear` will not match animals whose name contains `ear` like `Bear`, it will only search for animals whose names are exactly `ear`.
```

Comment:
Might be good to state that the same applies when searching based on other attributes! No partial match, only exact match